src/addrman.cpp

```diff
@@ -66,14 +66,16 @@ int AddrInfo::GetBucketPosition(const uint256& nKey, bool fNew, int nBucket) con
 
 bool AddrInfo::IsTerrible(int64_t nNow) const
 {
-    if (nLastTry && nLastTry >= nNow - 60) // never remove things tried in the last minute
+    if (nNow - nLastTry <= 60) { // never remove things tried in the last minute
```

Comment:
mega nit: fa8bdb7505e12afee4bc21e08ca17c36aaf7edcf calls it `m_last_try` but the member is still called `nLastTry`.

Reply:
Thx, fixed.

Reply:
@Riahiamirreza `nLastTry` is a plain integral value, so it can't be null or false. At most it can be `0`, but it seems odd to treat `0` (seconds) differently, from say `1` (seconds). `nNow - 0 <= 60` or `nNow - 1 <= 60` should be false either way.